about/async/suggest.js

```diff
@@ -53,11 +54,50 @@ exports.create = function (api) {
       suggestion,
       {idle: true}
     )
-    suggestions = map(contacts, suggestion, {idle: true})
+
+    const suggestionsRecord = computed(contacts, contacts => {
+      var result = {}
+      contacts.forEach(contact => {
+        result[contact] = api.about.obs.names(contact)
+      })
+
+      return result
+    })
+
+    const mapableSuggestions = dictToCollection(suggestionsRecord)
+    suggestions = concat(
+      map(mapableSuggestions, pluralSuggestions, {idle: true})
+    )
```

Comment:
This might be a little tidier doing the `dictToCollection` inline:

```js
suggestions = concat(
  map(dictToCollection(suggestionsRecord), pluralSuggestions, {idle: true})
)
``` 

eh, a matter of taste.